pkg/git/git.go

```diff
@@ -74,6 +77,12 @@ func GetSystemQualifiedPath(clusterName string, relativePath string) string {
 	return filepath.Join(GetSystemPath(clusterName), relativePath)
 }
 
+// GetProfilesPath returns the path of the file containing the manifests of installed Profiles
+// joined with the cluster's system directory
+func GetProfilesPath(clusterName string) string {
```

Comment:
This might be a thing that the notification stuff can use in order to notify only for installed profiles 🤔 Nice.